go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/iotexproject/iotex-proto
 go 1.21
 
 require (
+	github.com/golang/mock v1.6.0
```

Comment:
not needed?

Reply:
it's required by golang/iotexapi/mock_iotexapi/mock_iotexapi.go